rep-0003.rst

```diff
@@ -180,6 +179,46 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+- C++11
+
+- Python 2.7
+
+  - Python 3.4 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+- Boost 1.58
+
+Exact or Series Requirements:
+
+- Ogre3D 1.9.x
+- Gazebo 5 (TODO(esteve): or Gazebo 6?)
```

Comment:
I would say that the guideline was try to use the gazebo system version as much as we can, but it is always nice to provide ROS users with one of the latest versions of Gazebo when the release cycle starts. It is almost sure that 16.04 and 15.10 have different versions of gazebo so we will have to import packages anyway, hopefully only in one distribution.